cmake/config.cmake.in

```diff
@@ -18,6 +18,7 @@
 set(DNNL_CPU_RUNTIME "@DNNL_CPU_RUNTIME@")
 set(DNNL_CPU_THREADING_RUNTIME "@DNNL_CPU_THREADING_RUNTIME@")
 set(DNNL_GPU_RUNTIME "@DNNL_GPU_RUNTIME@")
+set(DNNL_BLAS_VENDOR "@DNNL_BLAS_VENDOR@")
```

Comment:
Hi @jacobkahn, thank you for the PR.

Could you elaborate, please, why this is needed? I'd expect that [this line](https://github.com/oneapi-src/oneDNN/blob/master/cmake/blas.cmake#L63) should satisfy the requirement...

Reply:
@dzarukin — this isn't needed; I just added it as a convenience for downstream projects. Shall I remove it?

Reply:
It'll be good to understand how does it help. Could you, please, explain? Thanks.

Reply:
I've removed it so we can merge the PR, but it might be helpful for a project that wants to conditionally compile based on what vendor oneDNN uses.

Consider a codepath that `mkl_set_num_threads(...)` in a downstream project. That project might not want to compile that codepath if DNNL is using MKL (for example, of the downstream project uses MKL directly) as it might interfere with DNNL. Likewise, a project might want to use a vendor lib which is consistent with the one DNNL was compiled with to reduce the number of deps, ensure consistency (use the same BLAS impls), etc.